composer.json

```diff
@@ -1,32 +1,36 @@
 {
     "name": "stadgent/services-opening-hours",
     "description": "Package to access the Opening Hours API and wrap the responses in value objects.",
-    "keywords": ["gent", "services", "opening hours"],
-    "homepage": "https://bitbucket.org/digipolisgent/php_package_gent-services-profile",
-    "type": "library",
     "license": "MIT",
+    "type": "library",
+    "keywords": [
+        "gent",
+        "services",
+        "opening hours"
+    ],
     "authors": [
         {
             "name": "Peter Decuyper",
             "email": "peter.decuyper@digipolis.gent"
         }
     ],
-    "minimum-stability": "dev",
-    "prefer-stable": true,
+    "homepage": "https://bitbucket.org/digipolisgent/php_package_gent-services-profile",
```

Comment:
where hosted on GitHub now (and on district09)

Reply:
```suggestion
    "homepage": "https://github.com/StadGent/php_package_services-opening-hours",
```